client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -90,6 +92,26 @@ const UserPage = createClass({
 		  </td>;
 	},
 
+	handleFilterTextChange : function(e){
+		this.setState({
+			filterString : e.target.value
+		});
+		return;
+	},
+
+	renderFilterOption : function(){
+		return <td>
+			<label>
+				<i className='fas fa-search'></i>
```

Comment:
`className` ??

Reply:
React - `<i className'x'>` renders to html as `<i class='x'>`.